core/payment/src/lib.rs

```diff
@@ -73,3 +75,30 @@ impl PaymentService {
         log::info!("Payment service stopped.");
     }
 }
+
+async fn release_allocation(db: DbExecutor) {
+    if let Ok(allocations) = db
+        .as_dao::<dao::AllocationDao>()
+        .get_filtered(None, None, None, None, None)
+        .await
+    {
+        allocations.iter().for_each(|a| {
+            let allocation = a.clone();
+            let db = db.clone();
+            let deadline = allocation.timeout.clone().unwrap_or(Utc::now());
```

Comment:
In case of timeout is set to `None` allocation is not released automatically at any point in time. 

The only way to release ALL the allocations is to use CLI command `yagna payment release-allocations` (already implemented). We can think about adding the option of releasing allocation by ID. Currently CLI command clear all the allocations.